boxoffice/views/admin.py

```diff
@@ -24,11 +25,27 @@ def index():
     return dict(user=g.user)
 
 
+def jsonify_ic(ic):
+    if ic:
+        return {
+            'id': ic.id,
+            'name': ic.name,
+            'title': ic.title,
+            'url': '/ic/' + unicode(ic.id),
+            'description_text': ic.description_text,
+            'description_html': ic.description_html
+        }
+    return None
```

Comment:
This line isn't required. `None` is returned by default.

Reply:
Fixed